custom_components/tuya_local/devices/blitzwolf_bw_lt31.yaml

```diff
@@ -0,0 +1,155 @@
+name: Blitzwolf BW-LT31 LED Strip
+products:
+  - id: 66072501f4cfa2fe2887
```

Comment:
This does not look like the usual format for a product id.  Is it really the product id, and not the device id?